apps/src/lib/util/RailsAuthenticityToken.jsx

```diff
@@ -7,16 +7,24 @@ import React from 'react';
  * This is necessary for React-rendered forms to submit to Rails endpoints
  * that have CSRF protection enabled.
  */
-export default function RailsAuthenticityToken() {
-  const csrfParam = document.querySelector('meta[name="csrf-param"]');
-  const csrfToken = document.querySelector('meta[name="csrf-token"]');
-  if (csrfParam && csrfToken) {
-    return (
-      <input type="hidden" name={csrfParam.content} value={csrfToken.content} />
-    );
+export default class RailsAuthenticityToken extends React.Component {
+  static getRailsCSRFMetaTags() {
+    let param, token;
+    const metaParam = document.querySelector('meta[name="csrf-param"]');
+    const metaToken = document.querySelector('meta[name="csrf-token"]');
+    if (metaParam && metaToken) {
+      param = metaParam.content;
+      token = metaToken.content;
+    } else {
+      console.error(
```

Comment:
This will get logged to someplace like NewRelic, right?

Reply:
Not that I know of; I just figured it would be helpful for a clear error message to show up in the console for developers.  I would have to call [`logToCloud.logError`](https://github.com/code-dot-org/code-dot-org/blob/3d86c78ad120d6e92bcaae27135b9bff0acb5bd0/apps/src/logToCloud.js#L95-L100) to notify New Relic.  Shall I add that here?